pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -546,6 +548,33 @@ func TestDeleteMultipleUserAccountsViaMasterUserRecordBeingDeleted(t *testing.T)
 		DoesNotHaveFinalizer()
 }
 
+func TestDisablingMasterUserRecord(t *testing.T) {
+	// given
+	logf.SetLogger(logf.ZapLogger(true))
+	s := apiScheme(t)
+	mur := murtest.NewMasterUserRecord("john", murtest.DisabledMur(true))
+	memberClient := test.NewFakeClient(t, consoleRoute(), cheRoute(false))
+	hostClient := test.NewFakeClient(t, mur)
+	cntrl := newController(hostClient, s, newGetMemberCluster(true, v1.ConditionTrue),
+		clusterClient(test.MemberClusterName, memberClient))
+
+	// when
+	res, err := cntrl.Reconcile(newMurRequest(mur))
+	require.NoError(t, err)
+	assert.Equal(t, reconcile.Result{}, res)
```

Comment:
@MatousJobanek it's when the useraccount is created. in the next reconcile we sync the spec and status. i guess it could be considered part of the setup

Reply:
so why don't you create the UserAccount as part of the test setup and add it to the fake client while initializing it - it should be pretty easy

Reply:
@MatousJobanek yeah, good idea! i've changed the test to do what you suggested!